### PR TITLE
SPLICE-1973 Exclude Kafka jars from splice-uber.jar for all platforms

### PR DESCRIPTION
**Pull request by stpabhi**

### CODE REVIEW

**Comment by OlegMazurov** [comment]
This PR conflicts with #1254 already in the queue. Let us integrate that one first.

File: assembly/pom.xml [hunk 1]
```diff
@@ -237,51 +238,69 @@
...
+            <properties>
+                <excludeSpark>org.apache.spark:*</excludeSpark>
+            </properties>
         </profile>
         <profile>
             <id>cdh5.8.3</id>
-            <properties><extraExclude>org.apache.spark:*</extraExclude></properties>
-        </profile>
-        <profile>
-            <id>cdh5.8.5</id>
```
Comment (OlegMazurov) [inline assembly/pom.xml]: Is 5.8.5 removed by mistake?
I'd rather see this whole part intact. All is needed is <exclude>org.apache.kafka:*</exclude> above.

Comment (stpabhi) [reply assembly/pom.xml]: @OlegMazurov must be my mistake. will revert this whole section

**Comment by stpabhi** [comment]
@OlegMazurov Done. please review it.

**Pull request merged**